src/main/kotlin/org/taktik/icure/services/external/rest/v2/controllers/extra/TimeTableController.kt

```diff
@@ -94,6 +94,7 @@ class TimeTableController(
 
 				//Create a timeTableItem
 				val timeTableItem = TimeTableItem(
+					rrule = null,
 					calendarItemTypeId = "consult",
 					days = mutableListOf("monday"),
```

Comment:
As they are already deprecated in V1, days & recurrenceTypes should be deleted from V2 and replaced by rrule instead